tests/convert_if_to_select/expected_outputs.mlir

```diff
@@ -0,0 +1,160 @@
+// RUN: heir-opt  %s | FileCheck %s
+
+// CHECK-LABEL: @secret_condition_with_non_secret_int
+func.func @secret_condition_with_non_secret_int(%inp: i16, %cond: !secret.secret<i1>) -> !secret.secret<i16> {
+  // CHECK-NEXT: %[[RESULT:.*]] = secret.generic ins(%[[INP:.*]], %[[COND:.*]] : [[T:.*]], !secret.secret<i1>) {
+  // CHECK-NEXT:   ^[[bb0:.*]](%[[CPY_INP:.*]]: [[T]], %[[SCRT_COND:.*]]: i1):
+  // CHECK-NEXT:     %[[ADD:.*]] = arith.addi %[[CPY_INP]], %[[CPY_INP]] : [[T]]
+  // CHECK-NEXT:     %[[SEL:.*]] = arith.select %[[SCRT_COND]], %[[ADD]], %[[CPY_INP]] : [[T]]
+  // CHECK-NEXT:     secret.yield %[[SEL]] : [[T]]
+  // CHECK-NEXT:   } -> !secret.secret<[[T]]>
+  // CHECK-NEXT: return %[[RESULT]] : !secret.secret<[[T]]>
+  %0 = secret.generic ins(%inp, %cond : i16, !secret.secret<i1>) {
+    ^bb0(%copy_inp: i16, %secret_cond: i1):
+      %2 = arith.addi %copy_inp, %copy_inp : i16
+      %1 = arith.select %secret_cond, %2, %copy_inp : i16
+      secret.yield %1 : i16
+    } -> !secret.secret<i16>
+    return %0 : !secret.secret<i16>
+}
+
+// -----
```

Comment:
Remove all of these from the (non-failing) tests, as they're only parsed by `mlir-opt`/`heir-opt` if you pass `-split-input-file`, and are only needed to test errors.
The `CHECK-LABEL` already does some of the work here in splitting test cases. If you want to incrase the isolation between test cases even further, you can add [the `--enable-var-scope` option](https://llvm.org/docs/CommandGuide/FileCheck.html#cmdoption-FileCheck-enable-var-scope) to the `FileCheck` arguments.